### ISSUE TITLE
Keyboard should be bigger with touch enlarge

### ISSUE THREAD

**Issue by chinmaynsamant** [chevyhu/Flysky-OpenTX-Test-Branch#38]
Touch enlarge feature enlarges everything but the keyboard. It will be very helpful if the keyboard becomes bigger like it does in UFPV.

**Comment by eshifri** [comment]
Looks like this functionality is intentionally disabled: line 132 in keyboard_text.cpp

**Comment by qba667** [comment]
@eshifri  you are right, alphabetic layout was criticized a lot. It must be redesigned to enlarged qwerty.

**Comment by eshifri** [comment]
@qba667 Thank you for explanation. Makes a lot of sense.
Doesn't it mean just not to use LARGE_KEYBOARD_... and use KEYBOARD_... always?
Then everything size-related will still can be used as-as?

**Comment by eshifri** [comment]
@qba667 Please disregard: I see what the problem is.

**Comment by chinmaynsamant** [comment]
I am not a C++ programmer so there is a possibility that I don't know what I am talking about. But can't we use the same piece of code from Undergroundfpv repository to enlarge the keyboard? When I compiled FW from that repo, I could get the keyboard to enlarge.

**Comment by eshifri** [comment]
There are some other changes in the code between FlySky and UFPV branches, but if you do not mind alphabetic keyboard layout, I can E-mail you the file that brings this functionality back. You will need to build the SW yourself of course. Or I can send you the compiled FW (of course, no warranty of any kind).

**Comment by chinmaynsamant** [comment]
Thanks a lot for your help. I am satisfied with the current Keyboard (QWERTY is better). I was just reporting the issue. Once it is solved, it can be uploaded to the repository. I have made the setup on my PC to compile the FW. There is no rush.

**Comment by qba667** [comment]
@eshifri hat we can do is increase button height - there is no space for width change and keeping qwerty.
Currently I always use standard keyboard - change I made is moving it top a little to get easier access to space.

**Comment by eshifri** [comment]
@qba667 Thank you for the explanation again!
I have noticed these changes and I see the problem: no space for bigger buttons.
Unfortunately, I have to agree with chinmaynsamant: the smaller buttons are inconvenient. I tried to edit some names and I was touching wrong letters more often than not.
Unfortunately (again) I have no suggestions how to fix that. May be alphabetic order is not that bad after all? Small qwerty does not allow one to use any typing skills anyway: too small buttons, too many misses.
